ci/jenkins/test.sh

```diff
@@ -468,10 +462,59 @@ function deliver_antrea_windows {
         done
     done
     rm -f antrea-windows.tar.gz
+    echo "==== Finish building and delivering Windows Docker images ===="
 }
 
-function deliver_antrea_windows_containerd {
-    echo "====== Cleanup Antrea Installation Before Delivering Antrea Windows Containerd ======"
+function build_and_deliver_antrea_windows_and_linux_docker_images {
+    echo "====== Cleanup Antrea Installation Before Delivering Antrea Windows and Antrea Linux Docker Images ====="
+    clean_antrea
+    kubectl delete -f ${WORKDIR}/antrea-windows.yml --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/kube-proxy-windows.yml --ignore-not-found=true || true
+    kubectl delete daemonset antrea-agent -n kube-system --ignore-not-found=true || true
+    kubectl delete -f ${WORKDIR}/antrea.yml --ignore-not-found=true || true
+
+    prepare_env
+    ${CLEAN_STALE_IMAGES}
+    ${PRINT_DOCKER_STATUS}
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
+  
+    if [[ "$TESTCASE" == "windows-networkpolicy-process" ]]; then
+        make windows-bin
+    fi
+
+    export_govc_env_var
+
+    # Enable verbose log for troubleshooting.
+    sed -i "s/--v=0/--v=4/g" build/yamls/antrea.yml build/yamls/antrea-windows.yml
```

Comment:
I see a `--v=0` in `antrea-windows.yml`, but not in `antrea.yml`, so this `sed` command will have no effect for `antrea.yml`:

https://github.com/antrea-io/antrea/blob/f679100427d1b2ca2fb5be93c29181459e75b62e/build/yamls/antrea.yml#L6874-L6885

Maybe the argument got removed when we moved over to Helm to generate the Linux manifest.

Reply:
Yes, we could do that. It's probably useful to have more verbose logging when troubleshooting tests.
I feel like we should use helm to generate manifests with the correct parameters however, instead of relying on `sed` extensively. One issue is that we still do not use Helm to generate the windows yaml manifests.

Anyway, a quick patch for this is to just remove the conditional in https://github.com/antrea-io/antrea/blob/00109a3eb2331a1593ae6d7270e30a793b65b6ef/build/charts/antrea/templates/agent/daemonset.yaml#L141